flocker/node/functional/test_docker.py

```diff
@@ -138,15 +138,6 @@ def start_container(self, unit_name,
 
         return d
 
-    def test_default_base_url(self):
-        """
-        ``DockerClient`` instantiated with a default base URL for a socket
-        connection has a client HTTP url after the connection is made.
-        """
-        client = DockerClient()
-        self.assertEqual(client._client.base_url,
-                         u'http+unix://var/run/docker.sock')
-
```

Comment:
I'm just deleting the test, since we don't really care if docker-py does with `base_url` if we don't specify one.